site/app/libraries/database/PostgresqlDatabaseQueries.php

```diff
@@ -1219,8 +1265,20 @@ public function getGradeableDataAll(\app\models\gradeable\Gradeable $gradeable,
                 $submitter = new User($this->core, $row);
             }
 
-            // Create the graded gradeable instance
-            $graded_gradeable = new GradedGradeable($this->core, $gradeable, new Submitter($this->core, $submitter), $row);
+            // Create the graded gradeable instances
+            $graded_gradeable = new GradedGradeable($this->core, $gradeable, new Submitter($this->core, $submitter));
+            $ta_graded_gradeable = null;
+            $auto_graded_gradeable = null;
+            if (isset($row['id'])) {
+                $ta_graded_gradeable = new TaGradedGradeable($this->core, $graded_gradeable, $row);
+                $graded_gradeable->setTaGradedGradeable($ta_graded_gradeable);
```

Comment:
This is being created and set, even for Gradebles from `grades_released_homework_onlyauto` which specifically has no ta grading.

Reply:
Seems like this is because you SELECT FROM gradeable_data and id / `gd_id` is a primary key, so `$row["id"]` can never be null

Reply:
Actually it seems that's just a quirk of the sample course. Deleting the gradeable_data a row seems to load this correctly. Still confusing though